src/lib/components/collections/collection-tab/MyCollection.viewModel.ts

```diff
@@ -29,40 +29,48 @@ export class MyCollectionViewModel {
 
   public modifyCollection = async (
     componentData,
-    newCollectionName: string,
+    property: string,
+    value: string,
     collectionsMethods: CollectionsMethods,
-    tabName: string,
   ) => {
-    if (newCollectionName) {
-      const updateCollectionElement =
-        await this.collectionService.updateCollectionData(
-          componentData.path.collectionId,
-          componentData.path.workspaceId,
-          { name: newCollectionName },
-        );
+    const updateObj = {};
 
-      tabName = updateCollectionElement.data.data.name;
-
-      collectionsMethods.updateCollection(
+    switch (property) {
+      case "name": {
+        updateObj.name = value;
```

Comment:
use same -> updateObj[property] = value;

Reply:
Correct. How did this not come to my mind 🤣